clr-k8s-examples/create_stack.sh

```diff
@@ -96,23 +99,40 @@ function kata() {
 }
 
 function cni() {
-	# note version is not semver
-	CANAL_VER=${1:-$CANAL_VER}
-	CANAL_URL="https://docs.projectcalico.org/${CANAL_VER}/manifests"
-	if [[ "$CANAL_VER" == "v3.3" ]]; then
-		CANAL_URL="https://docs.projectcalico.org/v3.3/getting-started/kubernetes/installation/hosted/canal"
-	fi
-	CANAL_DIR="0-canal"
-
-	# canal manifests are not kept in repo but in docs site so use curl
-	mkdir -p "${CANAL_DIR}/overlays/${CANAL_VER}/canal"
-	curl -o "${CANAL_DIR}/overlays/${CANAL_VER}/canal/canal.yaml" "$CANAL_URL/canal.yaml"
-	if [[ "$CANAL_VER" == "v3.3" ]]; then
-		curl -o "${CANAL_DIR}/overlays/${CANAL_VER}/canal/rbac.yaml" "$CANAL_URL/rbac.yaml"
-	fi
-	# canal doesnt pass kustomize validation
-	kubectl apply -k "${CANAL_DIR}/overlays/${CANAL_VER}" --validate=false
+	case "$CLRK8S_CNI" in
```

Comment:
Remove the CLRK8S_ as its is causing undeclared var error in bash.

Reply:
Done